src/net/rpc.rs

```diff
@@ -268,11 +268,36 @@ impl RPCPoxInfoData {
     ) -> Result<RPCPoxInfoData, net_error> {
         let mainnet = chainstate.mainnet;
         let chain_id = chainstate.chain_id;
-        let contract_identifier = boot_code_id("pox", mainnet);
+
+        let current_burn_height = chainstate
+            .maybe_read_only_clarity_tx(&sortdb.index_conn(), tip, |clarity_tx| {
```

Comment:
Can you use `with_read_only_clarity_tx` instead?  `maybe_read_only_clarity_tx` queries the unconfirmed microblock chain, which is unnecessary here.

Reply:
Done!